docs/sql-ref-syntax-ddl-repair-table.md

```diff
@@ -37,6 +37,13 @@ MSCK REPAIR TABLE table_identifier
 
     **Syntax:** `[ database_name. ] table_name`
 
+* **`[[ADD|DROP|SYNC] PARTITIONS]`**
+
+    * If the option is not specified, `MSCK REPAIR TABLE` adds partitions to the Hive external catalog only.
+    * **ADD**, the command adds new partitions in the catalog for all sub-folder in the base table folder that don't belong to any table partitions.
```

Comment:
Like line 42 (before this line) and line 44 (after this line), `the catalog` -> `the Hive external catalog` consistently?

Reply:
Actually, the command calls the session catalog which is an internal catalog servers as a proxy to an external catalog (In-Memory or Hive). Strictly speaking, the command adds/drops partitions in the session catalog. Let me update the doc to be more precise.